app/services/ember-cli.js

```diff
@@ -395,6 +395,13 @@ export default Ember.Service.extend({
       </script>`;
     }
 
+    depScriptTags += `
+      <script type="text/javascript">
+        Object.keys(requirejs.entries).forEach(function (moduleName) {
+          require(moduleName).__esModule = true;
```

Comment:
Hmm, I don't think you should need to do this...

Reply:
Added comment explaining